concurrent-map-bench does not compile

Building `concurrent-map-bench` in `benchmark` results in:

```
$ cargo build --release
...
error[E0599]: no method named `get_and` found for reference `&evmap::ReadHandle<u64, u64>` in the current scope
   --> src/main.rs:228:15
    |
228 |             r.get_and(&key, |v| v[0]).unwrap_or(0)
    |               ^^^^^^^ method not found in `&evmap::ReadHandle<u64, u64>`
```

I think this is caused by the API change in a55390c.

---

When I build the crate with cargo, I see the `evmap` dependency updated in `Cargo.lock` (9.0 vs. the committed 7.0). I think this is expected for a local `path` dependency, but please let me know if I'm doing something unexpected (I'm new to Rust).

If you can confirm that this is an issue, I'd be happy to create a PR with a fix.

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -3,6 +3,14 @@ jobs:
    parameters:
      minrust: 1.40.0
      codecov_token: $(CODECOV_TOKEN_SECRET)
+ - job: benchmark
+   displayName: "Check that benchmark compiles"
+   pool:
+     vmImage: ubuntu-latest
+   steps:
+     - template: install-rust.yml@templates
+     - bash: cd benchmark && cargo check
+       displayName: cargo check benchmark/
  - job: miri
    displayName: "Run miri on test suite"
    pool:

--- a/benchmark/Cargo.toml
+++ b/benchmark/Cargo.toml
@@ -4,7 +4,7 @@ version = "0.1.0"
 authors = ["Jon Gjengset <jon@thesquareplanet.com>"]
 
 [dependencies]
-evmap = { path = "../" }
+evmap = { version = "9", path = "../" }
 chashmap = "2.1.0"
 clap = "2.20.3"
 zipf = "0.2.0"

--- a/benchmark/src/main.rs
+++ b/benchmark/src/main.rs
@@ -2,17 +2,17 @@ extern crate chashmap;
 #[macro_use]
 extern crate clap;
 extern crate evmap;
+extern crate parking_lot;
 extern crate rand;
 extern crate zipf;
-extern crate parking_lot;
 
 use chashmap::CHashMap;
-use std::collections::HashMap;
 use clap::{App, Arg};
+use std::collections::HashMap;
 
-use std::time;
 use std::sync;
 use std::thread;
+use std::time;
 
 fn main() {
     let matches = App::new("Concurrent HashMap Benchmarker")
@@ -70,8 +70,8 @@ fn main() {
     let dur_in_s = dur_in_ns as f64 / 1_000_000_000_f64;
     let span = 10000;
 
-    let stat =
-        |var: &str, op, results: Vec<(_, usize)>| for (i, res) in results.into_iter().enumerate() {
+    let stat = |var: &str, op, results: Vec<(_, usize)>| {
+        for (i, res) in results.into_iter().enumerate() {
             println!(
                 "{:2} {:2} {:10} {:10} {:8.0} ops/s {} {}",
                 readers,
@@ -82,7 +82,8 @@ fn main() {
                 op,
                 i
             )
-        };
+        }
+    };
 
     let mut join = Vec::with_capacity(readers + writers);
     // first, benchmark Arc<RwLock<HashMap>>
@@ -101,7 +102,8 @@ fn main() {
             let dist = dist.to_owned();
             thread::spawn(move || drive(map, end, dist, true, span))
         }));
-        let (wres, rres): (Vec<_>, _) = join.drain(..)
+        let (wres, rres): (Vec<_>, _) = join
+            .drain(..)
             .map(|jh| jh.join().unwrap())
             .partition(|&(write, _)| write);
         stat("std", "write", wres);
@@ -124,7 +126,8 @@ fn main() {
             let dist = dist.to_owned();
             thread::spawn(move || drive(map, end, dist, true, span))
         }));
-        let (wres, rres): (Vec<_>, _) = join.drain(..)
+        let (wres, rres): (Vec<_>, _) = join
+            .drain(..)
             .map(|jh| jh.join().unwrap())
             .partition(|&(write, _)| write);
         stat("chashmap", "write", wres);
@@ -149,7 +152,8 @@ fn main() {
             let dist = dist.to_owned();
             thread::spawn(move || drive(map, end, dist, true, span))
         }));
-        let (wres, rres): (Vec<_>, _) = join.drain(..)
+        let (wres, rres): (Vec<_>, _) = join
+            .drain(..)
             .map(|jh| jh.join().unwrap())
             .partition(|&(write, _)| write);
 
@@ -225,7 +229,9 @@ enum EvHandle {
 impl Backend for EvHandle {
     fn b_get(&self, key: u64) -> u64 {
         if let EvHandle::Read(ref r) = *self {
-            r.get_and(&key, |v| v[0]).unwrap_or(0)
+            r.get(&key)
+                .map(|v| v.iter().next().cloned().unwrap())
+                .unwrap_or(0)
         } else {
             unreachable!();
         }

